### PR TITLE
Example pull request

### PR DESCRIPTION
**Pull request by rmk135**
Example text!

### CODE REVIEW

**Comment by rmk135** [comment]
Example comment!

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM python:3.9-buster
+
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONPATH="${PYTHONPATH}:/code"
+
```
Comment (VovkVera) [inline Dockerfile:5]: What do you mean?

Comment (rmk135) [reply Dockerfile:5]: test-test

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM python:3.9-buster
+
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONPATH="${PYTHONPATH}:/code"
```
Comment (VovkVera) [inline Dockerfile:4]: Text comment

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+WORKDIR /code
+COPY . /code/
+
+RUN pip install -r requirements.txt
```
Comment (VovkVera) [inline Dockerfile:9]: text

**Review by VovkVera — APPROVED** [review]
Common comment

**Pull request merged**